src/app/design/frontend/base/default/template/fontis/australia/checkout/onepage.phtml

```diff
@@ -22,6 +22,7 @@
         var australiaCountryRegions = <?php echo $this->helper('directory')->getRegionJson(); ?>;
         var addressSuggestTemplate = <?php echo Zend_Json::encode($this->getChildHtml('fontis.australia.address.suggestion')); ?>;
         var addressFailureTemplate = <?php echo Zend_Json::encode($this->getChildHtml('fontis.australia.address.failure')); ?>;
+        var validateAddressById = parseInt('<?php echo Mage::getStoreConfig('fontis_australia/address_validation/enabled_by_id'); ?>');
```

Comment:
@ProxiBlue , `addresspotentialsTemplate` is also missing, but that might be specific to your project?